migrations/files.js

```diff
@@ -0,0 +1,225 @@
+/*eslint no-console: 0 */
+const chalk = require('chalk');
+
+const ran = false; // set to true to exclude migration
```

Comment:
could we use a schema version or something like this to decide if the migration is necessary on the fly?